v1.1/array.md

```diff
@@ -0,0 +1,64 @@
+---
+title: ARRAYS
+summary: The ARRAYS data type stores 
+toc: false
+---
+
+The `ARRAY` data type creates one-dimensional, 1-indexed, homogenous arrays of any non-array [data type](data-types.html).
```

Comment:
@jseldess Yes, that's what we mean. I assume readers would know what it means, since we don't elaborate on it in the value expressions docs either. Thoughts, @justinj ?